packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -504,8 +512,11 @@ class FlutterTask extends BaseFlutterTask {
 
     @InputFiles
     FileCollection getSourceFiles() {
-        File dependenciesFile = getDependenciesFile()
-        FileCollection sources = readDependencies(dependenciesFile)
+        FileCollection dependenciesFile = getDependenciesFiles()
+        FileCollection sources = project.files()
```

Comment:
Is this `project.files()` necessary because dependency files is missing some sources?

Reply:
Necessary since we're now looping over and appending to the sources list for each file in `getDependenciesFiles()`.